### PR TITLE
Changes to python tests

### PR DESCRIPTION
**Pull request by apaaramazon**
changes files test1.py and test2.py

### CODE REVIEW

File: test1.py [hunk 1]
```diff
@@ -16,13 +16,23 @@
 del paras['tol']
 print paras
 
+#tuples -- immutable
+lis = (19, 73, -42)
```
Comment (apaars) [inline test1.py]: Add one more number here.

File: test1.py [hunk 2]
```diff
@@ -16,13 +16,23 @@
...
+
 
 paras = {}
 paras['niter'] = 500
 paras['tol'] = 1.e-06
 for key in paras:
     print "a[%s] = %s" % (key,paras[key])
 
+#sets -- mutable, no order for elements
+myset = set()
```
Comment (apaars) [inline test1.py:32]: add "physics" also

File: test2.py
Comment (apaars) [inline test2.py:1]: Looks good, thanks @apaaramazon !

**Review by apaars — CHANGES_REQUESTED** [review]
PTAL

File: test1.py (hunk 2 above)
Comment (apaaramazon) [reply test1.py:32]: This is already done below. Does that look good?

File: test1.py (hunk 1 above)
Comment (apaaramazon) [reply test1.py]: Done.

**Comment by apaaramazon** [comment]
Ready for another review please.

File: test1.py (hunk 2 above)
Comment (apaars) [reply test1.py:32]: Yes!

**Pull request merged**